.evergreen/config/templates/test/load-balanced.yml

```diff
@@ -0,0 +1,17 @@
+tasks:
+  - name: "test-mongodb-%mongodbVersion%-loadbalanced"
+    tags: ["loadbalanced", "local", "%mongodbVersion%"]
+    commands:
+      - func: "bootstrap mongo-orchestration"
+        vars:
+          TOPOLOGY: "sharded_cluster"
+          MONGODB_VERSION: "%mongodbVersion%"
+          LOAD_BALANCER: "true"
+          SSL: "yes"
+      - func: "start load balancer"
+      - func: "start kms servers"
+      - func: "set aws temp creds"
+      - func: "run tests"
+        vars:
+          MONGODB_URI: "${SINGLE_MONGOS_LB_URI}"
```

Comment:
Do you care about preserving the original comment?

```
# Note: loadBalanced=true should already be appended to SINGLE_MONGOS_LB_URI
```

I'm indifferent, just asking if it was intentionally removed.

Reply:
It was removed on purpose as it didn't add any value IMO.